processor/cumulativetodeltaprocessor/processor.go

```diff
@@ -29,7 +29,13 @@ import (
 const enableHistogramSupportGateID = "processor.cumulativetodeltaprocessor.EnableHistogramSupport"
 
 func init() {
-	featuregate.GetRegistry().MustRegisterID(enableHistogramSupportGateID, featuregate.StageBeta, featuregate.WithRegisterDescription("Enables histogram conversion support"))
+	featuregate.GetRegistry().MustRegisterID(
+		enableHistogramSupportGateID,
+		featuregate.StageBeta,
+		featuregate.WithRegisterDescription("Enables histogram conversion support"),
+		featuregate.WithRegisterReferenceURL("https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/15288"),
```

Comment:
@TylerHelmuth do we have a reference issue for this (and if not, should we have one?)

Reply:
@mx-psi here is the original issue: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12423.  Here is the issue that is currently blocking removal of the featuregate: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15658.

I think I can work on that issue now tho since 0.66.0 came out and pdata was updated to allow clearing min/max.

Reply:
Thanks! I think we should use that issue instead @MovieStoreGuy, what do you think?

Reply:
I will update it to be: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15658

Just make sure to add the `featuregate.StageStable` once you want to remove it in the next issue so it will error for users trying to modify the feature gate.